tests/test_upload_client.py

```diff
@@ -38,9 +35,39 @@ def parse_graphql_query(query: str) -> Optional[QueryParts]:
         return None
 
     return QueryParts(
-        parts.group("node_type"),
-        parts.group("node_id"),
-        parts.group("fields"),
+        parts.group("node_type"), parts.group("node_id"), parts.group("fields"),
+    )
+
+
+@pytest.fixture
+def mock_simple_upload_client(mock_dir_path):
+    return client.GDCUploadClient(
+        token="dummy",
+        processes=2,
+        server="localhost",
+        upload_part_size=FIVE_MB,
+        multipart=False,
+        files=[
+            {"id": "file-id-2", "path": mock_dir_path},
+            {"id": "file-id-1", "project_id": "GDC-MISC", "path": mock_dir_path},
+        ],
+        debug=False,
+    )
+
+
+@pytest.fixture
+def mock_mp_upload_client(mock_dir_path):
```

Comment:
Could you expand "mp"?  This stands for "multipart", right?   Same for the tests with "mp" in their name.

I can guess that mp == multipart in the context of this PR.  But it'd be harder to figure that without the context.

Reply:
Yes, i was following the convention that was already there when they were using "mp" to signify multipart. I can expand them to multipart